### ISSUE TITLE
Training loss increases after several epochs

### ISSUE THREAD

**Issue by Wi-sc** [facebookresearch/ss3d#6]
Hi @kalyanvasudev,

Thanks for your exciting work and codes. I'm very interested in it and trying to apply it to another dataset (ShapeNet rendered by [DVR](https://github.com/autonomousvision/differentiable_volumetric_rendering.git)). But I met a weird problem and want to ask here if you have any ideas to locate the bugs. I know this is not your duty but I would really appreciate it if you could point out some plausible reasons.

I find the training loss decreases in the first epochs, and then increases. And the volume rendering mask and RGB images look correct in the first epochs, but then as the training loss increases the output mask and images are all black.

Here is training loss.
![image](https://user-images.githubusercontent.com/32695851/211144712-4286fcac-eb47-421c-ad67-cd3adbe4fae1.png)

Here are rendering results in the first epochs.
![image](https://user-images.githubusercontent.com/32695851/211144746-7cb47d49-40a4-420b-b811-1391673b6c6a.png)

Here are rendering results after 15 epochs.
![image](https://user-images.githubusercontent.com/32695851/211144795-7f80d42e-7331-48cb-b4df-44d6e15f09e8.png)

Thanks again.
